contracts/0.4.24/oracle/OracleChiCaller.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-FileCopyrightText: 2020 Lido <info@lido.fi>
+
+// SPDX-License-Identifier: GPL-3.0
+
+/* See contracts/COMPILERS.md */
+pragma solidity 0.4.24;
```

Comment:
no need to use outdated solidity in a standalone contract. we use 0.4.24 because we need to be compatible with aragon

Reply:
Agree. I opted to use 0.2.24 because I needed to import `interfaces/ILidoOracle.sol`. On the other hand, I may explicitly double it here, like I did with `interface IFreeFromUpTo` below.